### ISSUE TITLE
Righthand sidebar minimap navigation has buggy collapse/expand behaviour

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#973]
The sidebar nav buttons appear as blank buttons to a screen reader. The text is toggling between display:none and display:inline.

We can either include duplicate .sr-only text, or instead of display:none, use an absolute offscreen position. (The latter seems more efficient.)

(Is it useful to compare how we did the skip links to display on focus?)

**Comment by jkeck** [comment]
https://github.com/sul-dlss/SearchWorks/blob/master/app/views/catalog/record/_record_side_nav.html.erb

https://github.com/sul-dlss/SearchWorks/blob/master/app/views/catalog/record/_marc_metadata_sections.html.erb#L2-L17

**Comment by aeschylus** [comment]
The problem is with [this hover event](https://github.com/sul-dlss/SearchWorks/blob/73bf7b875cc15b886679c4a3a9744ea43eaa2f96/app/assets/javascripts/jquery.record-side-nav.js#L59-L61). Scrolling also closes it. And there doesn't seem to be any css expanding the buttons.

I can fix this pretty easily if I can get Searchworks running again.

**Comment by aeschylus** [comment]
See #1098

**Comment by aeschylus** [comment]
Should be closed by https://github.com/sul-dlss/SearchWorks/pull/1149.

**Comment by aeschylus** [comment]
Closed by PR #1149
